.github/scripts/runner_determinator.py

```diff
@@ -377,8 +393,18 @@ def get_runner_prefix(
             )
             enabled = True
         elif experiment_settings.rollout_perc:
+            do_check = True
+            if check_experiments:
+                if experiment_name not in check_experiments:
+                    exp_list = ", ".join(check_experiments)
+                    log.info(f"Skipping experiment '{experiment_name}', as it is not in the check_experiments list: {exp_list}")
+                    do_check = False
+            elif not experiment_settings.default:
```

Comment:
if check experiments is enabled, then we should ignore the `default` setting and only use the `check` experiments settings.

Else you'll pull in the `lf` prefix as well